CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 * [#4444](https://github.com/bbatsov/rubocop/pull/4444): Make `Style/Encoding` cop enabled by default. ([@deivid-rodriguez][])
 * [#4452](https://github.com/bbatsov/rubocop/pull/4452): Add option to `Rails/Delegate` for enforcing the prefixed method name case. ([@klesse413][])
+* Add missing examples in `Layout` cops documentation. ([@haziqhafizuddin][])
```

Comment:
We generally only add changelog entries for changes that affect behaviour for the end user. 🙂